app/controllers/issues_controller.rb

```diff
@@ -50,6 +50,7 @@ def followees
 
   def create
     @issue = current_user.issues.new(issues_params)
+    @issue = delete_empty_images(@issue)
```

Comment:
Also, it is look strange for me, firstly you build wrong issue, and after correct it.
Maybe try to assign already corrected issue?
You can pass params into method, and create issue there, delete epmty images from it, and return correct issue.